src/__tests__/reducers/rating/ratingReducers.test.js

```diff
@@ -0,0 +1,42 @@
+import createRatingReducer from '../../../reducers/rating/createRatingReducer';
+import initialState from '../../../store/initialStates/ratingInitialState';
+import { ratingActionsTypes } from '../../../actions-types';
+// import articles from '../../__mocks__/articles';
+// import article from '../../__mocks__/article';
+// import message from '../../__mocks__/articleMessage';
+// import errors from '../../__mocks__/errors';
+
```

Comment:
Please make sure that all comments are removed

Reply:
remove comments